packages/grafana-data/src/dataframe/MutableDataFrame.ts

```diff
@@ -231,11 +231,6 @@ export class MutableDataFrame<T = any> implements DataFrame, MutableVector<T> {
           field.parse = makeFieldParser(val, field);
         }
         val = field.parse(val);
-      } else if (field.type === FieldType.time && isArray(val)) {
-        if (!field.parse) {
-          field.parse = (val: any[]) => val[0] || undefined;
```

Comment:
@ryantxu can you check whether this automatic unboxing is needed somewhere else I did not find anything so far.

Reply:
could not find any